ui/scripts/ui/widgets/listView.js

```diff
@@ -860,6 +860,8 @@
         if (groupableColumns) {
             $tr.addClass('groupable-header-columns').addClass('groupable-header');
             $.each(fields, function(key) {
```

Comment:
This was broken in existing code and needed fixing. If an upper field is hidden we must not draw it's containing columns.